README.md

```diff
@@ -1 +1 @@
-# to-do-list
\ No newline at end of file
+# Installing-webpack
```

Comment:
- [ ] As suggested by the previous reviewer, your README file is not professional as it doesn't have contents that describe your project. Microverse has provided a [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) that you should use for all your projects 😎 Please use it and customize it to your project, Make sure that all sections are covered 🏃